src/khoj/routers/helpers.py

```diff
@@ -185,6 +189,42 @@ async def aget_relevant_information_sources(query: str, conversation_history: di
         return [ConversationCommand.Default]
 
 
+async def aget_relevant_modes(query: str, conversation_history: dict):
```

Comment:
nit: We should make the function name more specific

```suggestion
async def aget_relevant_output_mode(query: str, conversation_history: dict):
```